data/nlu/nlu.md

```diff
@@ -3934,9 +3946,8 @@
 - please tell steps for installing chatbot
 - what do I need to install Rasa
 - what sould i do to install rasa
+- installation steps of rasa
 - how can I install [rasa open source](product)?
-- The [open source](product) version
-- what is [rasa open source](product) used for?
```

Comment:
@melindaloubser1 it looks like you added these, was that an accident?

Reply:
I think  I added them because for the getting started flow these were input as answers to "which one do you want to know more about?" and for that to work the product has to be extracted

Reply:
yes but i don't think it should be part of the installation intent :D

Reply:
eek no they should not! my bad